engines/zypper_auth/lib/zypper_auth/engine.rb

```diff
@@ -48,7 +48,7 @@ def verify_instance(request, logger, system)
         #{details.join(', ')}
       LOGMSG
 
-      Rails.cache.write(cache_key, false, expires_in: 2.minutes)
+      Rails.cache.write(cache_key, false, expires_in: 20.minutes)
```

Comment:
This cache writing call is for instance verifications that have failed, do we want to keep failed ones for 20 min on cache ?